pkg/datapath/loader/loader.go

```diff
@@ -99,11 +104,57 @@ func removeEndpointRoute(ep datapath.Endpoint, ip net.IPNet) error {
 	})
 }
 
+func (l *Loader) reloadHostDatapath(ctx context.Context, ep datapath.Endpoint, objPath string) error {
+	symbols := [4]string{symbolToHostEp2, symbolFromHostEp2, symbolFromHostEp1, symbolToHostEp1}
+	directions := [4]string{dirIngress, dirEgress, dirIngress, dirEgress}
+
+	interfaceNames := make([]string, 2, 4)
+	if option.Config.IsFlannelMasterDeviceSet() {
+		interfaceNames[0] = option.Config.FlannelMasterDevice
+	} else {
+		interfaceNames[0] = option.Config.HostDevice
+	}
+	interfaceNames[1] = interfaceNames[0]
+	if option.Config.Device != "undefined" {
+		interfaceNames = append(interfaceNames, option.Config.Device)
+		if option.Config.EnableNodePort || option.Config.EnableHostFirewall ||
+			!option.Config.InstallIptRules && option.Config.Masquerade {
```

Comment:
Given @brb 's outstanding PR I think it'd be safe to drop this line.

```suggestion
		if option.Config.EnableNodePort || option.Config.EnableHostFirewall {
```